src/sentry/integrations/models/external_actor.py

```diff
@@ -40,6 +40,7 @@ class ExternalActor(ReplicatedRegionModel):
             (ExternalProviders.GITHUB, "github"),
             (ExternalProviders.GITHUB_ENTERPRISE, "github_enterprise"),
             (ExternalProviders.GITLAB, "gitlab"),
+            (ExternalProviders.JIRA_SERVER, "jira_server"),
```

Comment:
Going to mull over how we can auto-assign this in the future instead of duplicating it.

Reply:
ohhhh yeah, the current design is so weird with the random numbers and provider strings 🫠